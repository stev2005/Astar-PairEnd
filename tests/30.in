AAAGCAAAACTCCTCCATTGAACCGATCCTTGATACTAGACCCGAGACCTTCTATCAGGGGGAGCATCACCCTGATAAAC
15
TTTTGAAATGGCGAGGATGCATA
TAGTAATATGTCGTCC
AGTACCAATGCTGTGGG
TGCTCTGGCG
TTTCTAG
ATAGATCTCTCTGTACATAGCGGACCCACACGTCATACGCCACTTCCACC
GTAGAGTCTTGCGTGCTTAA
ACATTCTAGGGCGACA
GATACCTTTGTTTTTATGTAGTGACTGCCTCTTAAACTAAGTTCGGCGCCG
ACTGACT
AACTACAGAAGGTGTCACTCGGAGTTATCACCAGCCTCTTCCGATCCAGCCATGTGCTCGT
A
ACCTGATTCCTCGACCCGCAAAGG
AATGATTCAGTACGGGCTCGAGTGTA
AGCAGCTCGTATTGTAACGGTCTCCCATCAG
