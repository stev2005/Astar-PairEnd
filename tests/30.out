9
6
7
4
2
22
9
6
24
2
30
0
9
12
13
